app/.storybook/main.js

```diff
@@ -4,87 +4,37 @@
  */
 // @ts-check
-const sassOptions = require("../scripts/sassOptions");
-
 // Support deploying to a subdirectory, such as GitHub Pages.
 const NEXT_PUBLIC_BASE_PATH = process.env.NEXT_PUBLIC_BASE_PATH ?? "";
-const storybookSassOptions = sassOptions(NEXT_PUBLIC_BASE_PATH);
 
 /**
- * @type {import("@storybook/core-common").StorybookConfig}
+ * @type {import("@storybook/nextjs").StorybookConfig}
  */
 const config = {
   stories: ["../stories/**/*.stories.@(mdx|js|jsx|ts|tsx)"],
   addons: ["@storybook/addon-essentials", "storybook-react-i18next"],
-  framework: "@storybook/react",
-  core: {
-    builder: {
-      name: "webpack5",
-      options: {
+  framework: {
+    name: "@storybook/nextjs",
+    options: {
+      builder: {
         // Cache build output between runs, to speed up subsequent startup times
         fsCache: true,
         // Applies in development mode. Storybook will start up faster, at the cost
         // of slightly slower browsing time when you navigate to another story.
         lazyCompilation: true,
       },
     },
+  },
+  core: {
     disableTelemetry: true,
   },
-  // Tell storybook where to find USWDS static assets
   staticDirs: ["../public"],
   // Support deploying Storybook to a subdirectory (like GitHub Pages).
   // This makes `process.env.NEXT_PUBLIC_BASE_PATH` available to our source code.
-  // @ts-expect-error - https://github.com/storybookjs/storybook/issues/19294
   env: (config) => ({
     ...config,
     NEXT_PUBLIC_BASE_PATH,
   }),
-  // Configure Storybook's final Webpack configuration in order to re-use the Next.js config/dependencies.
-  webpackFinal: (config) => {
-    config.module?.rules?.push({
-      test: /\.scss$/,
-      use: [
-        "style-loader",
-        { loader: "css-loader", options: { url: false } }, // this mirrors next.js behavior
-        {
-          /**
-           * Next.js sets this automatically for us, but we need to manually set it here for Storybook.
-           * The main thing this enables is autoprefixer, so any experimental CSS properties work.
-           */
-          loader: "postcss-loader",
-          options: {
-            postcssOptions: {
-              plugins: ["postcss-preset-env"],
-            },
-          },
-        },
-        {
-          loader: "sass-loader",
-          options: {
-            sassOptions: storybookSassOptions,
-          },
-        },
-      ],
-      exclude: /node_modules/,
-    });
-
-    // Required for i18next.
-    config.resolve = config.resolve ?? {};
-    config.resolve.fallback = {
-      fs: false,
-      path: false,
-      os: false,
-    };
-
-    // Support deploying Storybook to a subdirectory (like GitHub Pages).
-    // This makes the Storybook JS bundles load correctly.
-    if (NEXT_PUBLIC_BASE_PATH) {
-      config.output = config.output ?? {};
-      config.output.publicPath = `${NEXT_PUBLIC_BASE_PATH}/`;
-    }
-
-    return config;
```

Comment:
❓ Did you intend on removing this -- would this cause issues with deploying to github pages w/o it?

Reply:
Yeah, it seems like the Next.js Storybook framework integration now does all of this for us. I tested deploying this branch to GitHub Pages and it works: https://navapbc.github.io/template-application-nextjs/